src/set_theory/ordinal/arithmetic.lean

```diff
@@ -1449,9 +1484,12 @@ eq_of_forall_ge_iff $ λ o,
 by rw [blsub_le_iff, lsub_le_iff]; exact
   ⟨λ H b, H _ _, λ H i h, by simpa only [typein_enum] using H (enum r i h)⟩
 
-theorem blsub_const {o : ordinal} (ho : o ≠ 0) (a : ordinal) : blsub.{u v} o (λ _ _, a) = a + 1 :=
+theorem blsub_const {o : ordinal} (ho : o ≠ 0) (a : ordinal) : blsub.{u v} o (λ _ _, a) = a.succ :=
```

Comment:
Any reason for the change to `succ`?

Reply:
(no objection, but please explain in the PR description if you think it's an improvement)

Reply:
`a.succ` is def-eq to `a + 1`, but the former has more API.